.nsprc

```diff
@@ -6,6 +6,7 @@
     "https://nodesecurity.io/advisories/534",
     "https://nodesecurity.io/advisories/535",
     "https://nodesecurity.io/advisories/550",
-    "https://nodesecurity.io/advisories/532"
+    "https://nodesecurity.io/advisories/532",
+    "https://nodesecurity.io/advisories/146"
```

Comment:
We need to remove this one before merging into the master

Reply:
@NejcZdovc i thought this was going to be maintained as a separate branch, not merged into master?

Reply:
it still is in the `usermodel` branch, which we regularly rebase from master to stay current with "the latest and greatest", however, afaik, none of the `usermodel` branch has gone into master.